man/systemd.network.xml

```diff
@@ -1625,6 +1625,13 @@
           </listitem>
         </varlistentry>
 
+        <varlistentry>
+          <term><varname>PrefixDelegationHint=</varname></term>
+          <listitem>
+            <para>As in the "[Network]" section. Specifies the DHCPv6 client for the requesting router to include a prefix-hint in
```

Comment:
It is better to explain explicitly.
`As in the "[Network]" section.` -> `Takes an IPv6 address with prefix length separated with <literal>/</literal>.`

Reply:
That explains it . no need for extra explanation.